src/habitat.cr

```diff
@@ -2,14 +2,16 @@ require "./habitat/*"
 
 class Habitat
   class MissingSettingError < Exception
-    def initialize(type, setting_name)
+    def initialize(type, setting_name, example)
+      example = "some_value" if example == Nil
+      example = "\"#{example}\"" if example.is_a?(String)
```

Comment:
I think this might cause issues with more complex examples. For example, an example where the setting is created by calling another object: `settings.logger = MyCoolLogger.new`. I wouldn't want the `MyCoolLogger.new` wrapped in quotes. So I'm thinking this could be removed and we should document that if you want a string literal then you need quotes. 

I think this is ok since if you're ready to add an example you are probably looking at the docs for it as it is not as common of a feature. What do you think?